lighty-modules/lighty-openflow-sb/src/main/resources/defaultOfpConfig.json

```diff
@@ -7,7 +7,7 @@
     "enableFlowRemovedNotification": true,
     "skipTableFeatures": true,
     "basicTimerDelay": 3000,
-    "maximumTimerDelay": 3679,
+    "maximumTimerDelay": 900000,
```

Comment:
Is it OK to have maximumTimerDelay this big?

Reply:
Hmm yeah is harder to see it now. There was duplicate property with name nonZeroUint32Type. And this property was overriding maximumTimerDelay property in construction.
https://github.com/PANTHEONtech/lighty/pull/675/files/7521800919de874239c691bc48075cd41c24db9a#diff-b3861a128f6f9e37226de0209f6a20baf2e3aa8ceae31b82a472f3c8f7d6ec8a

So I removed that duplicate property with name nonZeroUint32Type.
```java
.setMaximumTimerDelay(new NonZeroUint32Type(this.nonZeroUint32Type))
``` 

But when "nonZeroUint32Type" property was overriding with his value the property "maximumTimerDelay" So I set "nonZeroUint32Type" value to  "maximumTimerDelay".

I don't know if this value is correct, but it was set like this before. But it was not seen.